src/main/java/com/iexec/common/docker/client/DockerClientInstance.java

```diff
@@ -164,24 +162,20 @@ public synchronized boolean removeVolume(String volumeName) {
         if (StringUtils.isBlank(volumeName)) {
             return false;
         }
-        if (!isVolumePresent(volumeName)) {
-            log.info("No docker volume to remove [name:{}]", volumeName);
-            return false;
-        }
         try (RemoveVolumeCmd removeVolumeCmd = getClient().removeVolumeCmd(volumeName)) {
             removeVolumeCmd.exec();
             log.info("Removed docker volume [name:{}]", volumeName);
             return true;
+        } catch (NotFoundException e) {
+            log.info("No docker volume to remove [name:{}]", volumeName);
```

Comment:
`log.error` everywhere*

Reply:
hmm, I just moved the logging events without modifying the level.
I don't know if we want an `error` here, maybe a `warn` ?
(For me, `error`, should be used to detect things that should not have happened, it is more to detect potential application failures)
I kind of like those descriptions
https://sematext.com/blog/logging-levels/

Reply:
We can start with a `warn` yes :+1: 
(We might have this conversation again since we still need to rework return statuses at some point)

Reply:
I agree there is room for discussion here.
Done in f776602